### PR TITLE
Develop to Main

### PR DESCRIPTION
**Pull request by dcolleoni**
Merging develop branch to main. 
This point corresponds to a deployment for clients.

### CODE REVIEW

File: ifrs17/Utils/ActivityLog.ipynb [hunk 1]
```diff
@@ -48,6 +66,25 @@
...
+            ],
+            "metadata": {},
+            "execution_count": 0,
+            "outputs": []
+        },
+        {
+            "cell_type": "markdown",
+            "source": [
+                "## Import Export Activity ",
+                "\nEvery activity to and from (import or export) the DataSource is tracked by recording an ImportExportActivity. Among other information this record tracks the user who performe the action, the timestamp, the activity log, and a unique identifier which can be used to retrieve the data subject of the activity. "
```
Comment (nnikolopoulos) [inline ifrs17/Utils/ActivityLog.ipynb:82]: Typo: performe -> performs (minor comment)

File: PresentValueSeries/InitializeData.ipynb [hunk 2]
```diff
@@ -108,4 +108,4 @@
             "outputs": []
         }
     ]
-}
```
Comment (nnikolopoulos) [inline PresentValueSeries/InitializeData.ipynb:111]: Unnecessary change (minor comment)

File: PresentValueSeries/PresentValue - Episode 2.ipynb [hunk 3]
```diff
@@ -507,4 +507,4 @@
             "outputs": []
         }
     ]
-}
```
Comment (nnikolopoulos) [inline PresentValueSeries/PresentValue - Episode 2.ipynb:510]: Also unnecessary (minor comment)

**Review by nnikolopoulos — APPROVED** [review]

File: PresentValueSeries/InitializeData.ipynb (hunk 2 above)
Comment (dcolleoni) [reply PresentValueSeries/InitializeData.ipynb:111]: I agree but it's harmless.

File: PresentValueSeries/PresentValue - Episode 2.ipynb (hunk 3 above)
Comment (dcolleoni) [reply PresentValueSeries/PresentValue - Episode 2.ipynb:510]: I agree but it's harmless.
